packages/client/src/admin/SystemSetting/SystemSetting.tsx

```diff
@@ -423,7 +423,8 @@ function _SystemSetting({ data, ...props }: Props) {
                   <InputNumber
                     data-testid="settings-capacity"
                     defaultValue={defaultUserVolumeCapacity}
-                    formatter={(value) => `${value} GB`}
+                    formatter={(value) => `${value}GB`}
+                    parser={(value) => value.replace(/([A-Z]+.*)/, '')}
```

Comment:
The original input number component has default parser, you can just use it. https://github.com/InfuseAI/primehub-console/blob/master/packages/client/src/cms-components/customize-number-precision.tsx#L34